base.yaml

```diff
@@ -210,4 +210,12 @@ button:
     on_press: 
       then:
         lambda: !lambda |-
-          id($id_prefix).sync();
\ No newline at end of file
+          id($id_prefix).sync();
+
+  - platform: template
+    id: ${id_prefix}_toggle_door
+    name: "Toggle door"
+    on_press: 
+      then:
+        lambda: !lambda |-
+          id($id_prefix).toggle_door();
```

Comment:
For a long time I thought that HA `cover.toggle` service ends up calling esphome's cover toggle (`call.get_toggle()` set to true) if the cover traits has `set_supports_toggle(true)`. It turns out that's not the case, so I added a button for people that want simple one-button control of the garage door form HA.